### PR TITLE
List and sass

### PR DESCRIPTION
**Pull request by bednayb**

### CODE REVIEW

File: dist/index.html [hunk 1]
```diff
@@ -4,12 +4,10 @@
     <meta charset="utf-8">
     <title>BloodStone Bookmark Manager</title>
     <script src="bundle.js"></script>
-  <link href="https://fonts.googleapis.com/css?family=Libre+Baskerville:400,700" rel="stylesheet">
-  <link href="https://fonts.googleapis.com/css?family=Work+Sans" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css?family=Libre+Baskerville:400,700" rel="stylesheet">
```
Comment (hubudibu) [inline dist/index.html]: merge the 2 font links: `https://fonts.googleapis.com/css?family=Libre+Baskerville:400,700|Work+Sans`

File: dist/views/list.html [hunk 2]
```diff
@@ -1,13 +1,25 @@
...
-  <ul class = "linksContainer">
-    <li id="links" ng-repeat="link in dummyLinks"><p>Title:  {{link.title}}, Url: {{link.url}}</p></li>
-  </ul>
-
-  <h6>Team Zerda Lasers BloodStone, Green Fox Academy, 2017</h6>
-  <div class="plusURL">
-    <span class="popupBtn">+</span>
-  </div>
-</main>
+<section class= "listSite">
```
Comment (hubudibu) [inline dist/views/list.html:2]: remove excess whitespace, no space around `=`

File: dist/views/list.html [hunk 3]
```diff
@@ -1,13 +1,25 @@
...
+      <div class="small-Logo"></div>
+    </div>
+  </header>
+  <main class= "mainList">
+    <ul class = "linksContainer">
+      <li id="links" ng-repeat="link in dummyLinks"><p>Title:{{link.title}}, Url: {{link.url}}</p></li>
+    </ul>
+    <footer>
+      <div class="circle">
+        <div class="plusURL">
```
Comment (hubudibu) [inline dist/views/list.html]: pick one: URL or Url? I would suggest `URL`

File: src/scss/style.scss [hunk 4]
```diff
@@ -100,51 +95,145 @@ a:hover {
...
+  display: flex;
+  position: relative;
+  top: 12px;
+  border:none;
+  color: #360033;
+  opacity: 0.9;
+  font-size: 18px;
+  color:#360033
+}
+input.addUrlInput::-webkit-input-placeholder {         opacity: 0.5;
```
Comment (hubudibu) [inline src/scss/style.scss]: fix newline here

File: src/scss/style.scss [hunk 5]
```diff
@@ -100,51 +95,145 @@ a:hover {
...
+  height: 0;
+  border-left: 12px solid transparent;
+  border-right: 12px solid transparent;
+  border-top: 18px solid $background;
+  display: flex;
+  position: relative;
+  top: -205px;
+  left: 33px;
+}
+#triangle-shadow{
```
Comment (hubudibu) [inline src/scss/style.scss]: rather use classes instead of ids in css

File: src/scss/style.scss [hunk 6]
```diff
@@ -100,51 +95,145 @@ a:hover {
...
+  position: relative;
+  top: -218px;
+  left: 38px;
+  border-top: 18px solid rgba(0, 0, 0, 0.5);
+  z-index: -1;
+}
+.bigLogo{
+  width: 0;
+	height: 0;
```
Comment (hubudibu) [inline src/scss/style.scss:230]: fix indentation

File: src/scss/style.scss [hunk 7]
```diff
@@ -100,51 +95,145 @@ a:hover {
...
+.addUrlInput{
+  width: 280px;
+  height: 20px;
+  padding: 8px;
+  display: flex;
+  position: relative;
+  top: 12px;
+  border:none;
+  color: #360033;
+  opacity: 0.9;
```
Comment (hubudibu) [inline src/scss/style.scss]: use rgba() text color instead to keep white background

**Review by hubudibu — CHANGES_REQUESTED** [review]

**Review by bednayb — COMMENTED** [review]
small changes

**Review by hubudibu — APPROVED** [review]
👍

**Pull request merged**